src/async_impl/client.rs

```diff
@@ -1820,7 +1877,8 @@ impl PendingRequest {
 
         *req.headers_mut() = self.headers.clone();
 
-        *self.as_mut().in_flight().get_mut() = self.client.hyper.request(req);
+        *self.as_mut().in_flight().get_mut() =
```

Comment:
Should this match and use the same kind of client request as before?

Reply:
You're right. I'll fix this.